<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Task</string>
  </property>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Source File</string>
   </property>
   <widget class="QLineEdit" name="txt_source">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_source_browser">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Destination File</string>
   </property>
   <widget class="QLineEdit" name="txt_destination">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dest_browser">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_source_browser</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>browse_source()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>-204</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dest_browser</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>browse_destination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>-118</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>-42</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>-77</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_source()</slot>
  <slot>browse_destination()</slot>
  <slot>save()</slot>
 </slots>
</ui>
